service/awsconfig/v10/resource/ebsvolume/error.go

```diff
@@ -11,6 +12,20 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
+// IsVolumeNotFound asserts volume not found error from upstream's API code.
+func IsVolumeNotFound(err error) bool {
```

Comment:
Lets also have an error type for this matcher and assert against the cause.

Reply:
I don't follow this. There is already a volumeNotFoundError defined.

Reply:
Yes, in `service/awsconfig/v10/ebs/error.go`. This here is `service/awsconfig/v10/resource/ebsvolume/error.go`. One is the service, one is the resource.

Reply:
You can probably get one right and just copy paste.

Reply:
Ack. Thanks! The resource shouldn't need to care about the AWS error handling now its all in the ebs service. I'll get it sorted.

Reply:
Ok cool. So we can just drop this here?

Reply:
Yes, it's gone now.